ethportal-api/src/types/state_trie/account_state.rs

```diff
@@ -1,11 +1,23 @@
-use alloy_primitives::{B256, U256};
+use alloy_primitives::{keccak256, B256, U256};
 use alloy_rlp::{RlpDecodable, RlpEncodable};
+use serde::{Deserialize, Serialize};
 
 /// The Account State stored in the state trie.
-#[derive(Debug, Clone, PartialEq, Eq, RlpEncodable, RlpDecodable)]
+#[derive(Debug, Clone, PartialEq, Eq, RlpEncodable, RlpDecodable, Serialize, Deserialize)]
 pub struct AccountState {
     pub nonce: u64,
     pub balance: U256,
     pub storage_root: B256,
     pub code_hash: B256,
 }
+
+impl Default for AccountState {
+    fn default() -> Self {
+        Self {
+            nonce: 0,
+            balance: U256::ZERO,
+            storage_root: keccak256(alloy_rlp::encode([])),
```

Comment:
```suggestion
            storage_root: keccak256(alloy_rlp::EMPTY_STRING_CODE),
```
I think this should be the same